--- Makefile.PL~	2005-11-14 16:32:15.786434176 +0100
+++ Makefile.PL	2005-11-14 16:41:31.830902568 +0100
@@ -776,7 +776,7 @@
 	$extra =~ s/ {8}/\t/g;
 	$inherited .= $extra;
     }
-
+if ('i am' eq 'in a candid camera') {
     # Remove existing non-working 'uninstall' target.
     $inherited =~ s!^uninstall\s:.*$!!m
       or die "no uninstall target in: $inherited";
@@ -871,7 +871,7 @@
 
 };
     $inherited =~ s/\$location/$location/g or die;
-
+} # endif dorkcode
     return $inherited;
 }
 
